ubuntu-simple/README.md

```diff
@@ -175,18 +217,25 @@ with the addition of curl.
 First attach to your Docker container using the `CONTAINER` value we captured earlier
 
     sudo docker attach $CONTAINER
```

Comment:
And here you don't need `docker attach` anymore, just user `docker run -ti ...`.